### DISCUSSION TITLE
Changing Magic Mouse speed/scale from the app

### DISCUSSION THREAD

**Discussion by dabaranau** [rxhanson/Multitouch-Community#110] [Ideas]
At the moment Magic Mouse can't be configured properly in the Settings.
Maximum Speed of Apple's settings is not enough for me and also there is no way to disable adaptive acceleration.
Maybe it makes sense to add this ability to the app?

**Comment by rxhanson** [comment]
Thanks for the feedback/idea. The difficult thing with managing Apple's speed/acceleration with the Magic Mouse is that this is typically handled on layers in the OS that aren't exposed to 3rd parties (as far as I know). From my perspective, I wouldn't be able to make something "good" for this from the tools I have available to me.

**Reply by dabaranau** [reply]
Afaik there is workarounds with rewriting defaults for some values (like described here: https://gadgetstouse.com/blog/2021/06/01/increase-mouse-sensitivity-on-mac-beyond-max/) 
Maybe there is a way to just include solution something like this but inside UI as not so many people can use Terminal as tool?

**Reply by rxhanson** [reply]
Thanks for the link. Apple doesn't make any promises on behavior of terminal command values like this, and in the past has broken functionality that was configured via terminal command. From the perspective of Multitouch, it would make the app more fragile and it's hard for me to justify adding it in. Additionally, I haven't really found users that have been unable to use the Terminal to tweak settings; all of my apps have additional settings that can only be configured via Terminal command. I'm not trying to dismiss this idea, I am just cautious on what I want to add, and it has to be something that would benefit a lot of users, not bloat the UI, and not make the app fragile.

**Reply by dabaranau** [reply]
Thanks for answer. Hope it could be added in future
